httpserver/lmq_server.cpp

```diff
@@ -220,6 +294,12 @@ void LokimqServer::init(ServiceNode* sn, RequestHandler* rh,
     lokimq_->add_category("notify", lokimq::Access{lokimq::AuthLevel::none, false, false})
         .add_command("add_pubkey", [this](auto& m) { this->handle_notify_add_pubkey(m); })
         .add_command("get_subscriber_count", [this](auto& m) { this->handle_notify_get_subscriber_count(m); });
+
+
+    lokimq_->add_category("service", lokimq::Access{lokimq::AuthLevel::none, false, false})
+        .add_command("get_stats", [this](auto& m) { this->handle_get_stats(m); })
+        .add_command("get_logs", [this](auto& m) { this->handle_get_logs(m); });

```

Comment:
These really should be `add_request_command` instead since they always have an expected reply that the remote end will wait for.  Then change the actual handlers to use `message.reply("whatever")` instead of `lokimq_->send(conn, "whatever")`.